### ISSUE TITLE
Staging requires registration inversion

### ISSUE THREAD

**Issue by nrhorner** [mpi2/LAMA#10]
If skip_transform_inversion = true and staging = 'whole_embryo' for example it fails to find the inverted masks.

Make a check for this

**Comment by Dorky-Lever** [comment]
Yeah I've ran into this issue when running the test-code as well.

**Comment by Dorky-Lever** [comment]
I'm going to try implement whole embryo volume labelling using the otsu algorithm that Henrik gave to me a while back.  That way you can still the embryos without inversion.
